packages/lesswrong/components/quickTakes/QuickTakesEntry.tsx

```diff
@@ -126,8 +146,12 @@ const QuickTakesEntry = ({
     return null;
   }
 
+  const userIsNotApproved = !currentUser?.reviewedByUserId;
```

Comment:
This is true both if the user isn't approved, and also if the user is logged out